seed/README.md

```
@@ -0,0 +1,111 @@
+| `1a…`, `1b…` | interview slots, assignments |
+| `2a…`, `2b…` | self team allocation teams, assignments |
+| `3a…`, `3b…`, `3c…` | team allocation teams, skills, allocations |
+| `4a…`, `4b…`, `4c…` | presentation categories, slots, presentations |
+| `5a…` … `60…`, `6a…`, `6b…` | assessment schemas, categories, competencies, assessments, evaluations, feedback, action items |
+| `a…`, `b3…`, `d…` | the e2e fixture phases and participations (see `e2e/src/data/constants.ts`) |
+
+## The demo course
+
+`iPraktikumDemo` (`ios2526`) is the full-course example. Its graph is
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)